src/Datadog.Trace/Propagation/CompositePropagatorsProvider.cs

```diff
@@ -0,0 +1,62 @@
+using System;
```

Comment:
I don't see any unit test for that, should we add some here, or will it be separate PR?

Reply:
added tests